osf/management/commands/collection_submission_machine_state_backfill.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from osf.utils.workflows import CollectionSubmissionStates
+
+from osf.models import CollectionSubmission
+
+logger = logging.getLogger(__name__)
+
+
+def collection_submission_machine_state_backfill(*args, backfill_date=None, dry_run=False, **kwargs):
+    submissions = CollectionSubmission.objects.all()
+
+    logger.info(f'{submissions.count()} submissions to backfill')
+    if not dry_run:
+        if backfill_date:
+            submissions.filter(
+                date_created__lte=backfill_date,
+                collection__deleted__isnull=True,
+            ).update(
+                machine_state=CollectionSubmissionStates.ACCEPTED
+            )
+        else:
+            submissions.filter(
+                collection__deleted__isnull=True,
+            ).update(
+                machine_state=CollectionSubmissionStates.ACCEPTED
+            )
```

Comment:
Can we avoid this with some sneakiness in the migration? AddField with a default of ACCEPTED and then AlterField in the same migration with a default of IN_PROGRESS?